filters/include/pcl/filters/experimental/passthrough.h

```diff
@@ -0,0 +1,227 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2021-, Open Perception
+ *
+ *  All rights reserved
+ */
+
+#pragma once
+
+#include <pcl/filters/experimental/functor_filter.h>
+
+#include <limits> // for numeric_limits
+
+namespace pcl {
+namespace experimental {
+
+template <typename PointT>
+struct PassThroughFunctor {
+  PassThroughFunctor() = default;
+  PassThroughFunctor(const uindex_t* field_offset,
+                     const float* filter_limit_min,
+                     const float* filter_limit_max,
+                     const bool* has_filter_field,
+                     const bool* negative)
+  : field_offset_(field_offset)
+  , filter_limit_min_(filter_limit_min)
+  , filter_limit_max_(filter_limit_max)
+  , has_filter_field_(has_filter_field)
+  , negative_(negative)
+  {}
+
+  bool
+  operator()(const PointCloud<PointT>& cloud, index_t idx) const
+  {
+    // only filter out points with NaN at the XYZ fields (done in FunctorFilter),
+    // regardless if the filter behavior is inverted or not
```

Comment:
Needs a bit of work in the language. What about:
```
always filter out points with NaN in XYZ fields
regardless of the user's choice of what points should be filtered (normal/inverted)
```

Reply:
Done